Snakefile

```diff
@@ -26,10 +27,41 @@ rule validate:
     script:
         "scripts/validation.py"
 
+rule plot_validation:
+    params:
+        countries=config["validation"]["countries"],
+        clusters=config["validation"]["clusters"],
+        planning_horizon=config["validation"]["planning_horizon"],
+    output:
+        demand=RESULTS_DIR + "plots/" + "demand_validation_{clusters}_{countries}_{planning_horizon}.png",
+        capacity=RESULTS_DIR + "plots/" + "capacity_validation_{clusters}_{countries}_{planning_horizon}.png",
+        generation=RESULTS_DIR + "plots/" + "generation_validation_{clusters}_{countries}_{planning_horizon}.png",
+        generation_detailed=RESULTS_DIR + "plots/" + "generation_validation_detailed_{clusters}_{countries}_{planning_horizon}.png",
```

Comment:
Have you modified `demand_validation.py` script to read rule inputs and assign outputs?

Reply:
I think there is a mix up, there is no `demand_validation.py` script. Perhaps you meant `plots_validation.py`

Reply:
Looks like the error came from my end. I didn't commit the script until now. Sorry about that.